### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -22336,6 +22355,9 @@ pxoxies->proxies
 pxoxy->proxy
 pyhon->python
 pyhsical->physical
+pyhsically->physically
+pyhsicals->physicals
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:22359]: Niche but present: https://dictionary.cambridge.org/dictionary/english/physicals

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -9228,6 +9240,9 @@ deverloper->developer
 deverlopers->developers
 devestated->devastated
 devestating->devastating
+devfine->define
+devfined->defined
+devfines->defines
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:9245]: You could add the defining and definition(s) versions of this typo too

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -4189,6 +4193,7 @@ brwosing->browsing
 btye->byte
 btyes->bytes
 buad->baud
+bubbless->bubbles
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:4196]: Add the singular as an equivalent typo, bubble

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -1910,10 +1912,12 @@ appeares->appears
 appearning->appearing
 appeciate->appreciate
 appeded->appended
+appeding->appending
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:1915]: Add the apped typo too.

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -1537,6 +1537,8 @@ alternatievly->alternatively
 alternativey->alternatively
 alternativly->alternatively
 alternatve->alternative
+alternavtive->alternative
+alternavtives->alternatives
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:1541]: Add the alternatively version of this typo

**Review by peternewman — CHANGES_REQUESTED** [review]
A few more suggestions

**Review by bl-ue — APPROVED** [review]

**Pull request merged**